apps/storybook.namekit.io/stories/Namekit/TruncatedText.stories.tsx

```diff
@@ -0,0 +1,92 @@
+import { TruncatedText } from "@namehash/namekit-react";
+import type { Meta, StoryObj } from "@storybook/react";
+
+enum TruncatedTextExample {
+  SHORT_TEXT = "SHORT_TEXT",
+  LONG_TEXT = "LONG_TEXT",
+}
+
+const getExampleTruncatedText = (example: TruncatedTextExample): string => {
+  switch (example) {
+    case TruncatedTextExample.SHORT_TEXT:
+      return "heyiamsmall";
+    case TruncatedTextExample.LONG_TEXT:
+      return "heyiamaverylongtextthatkeepsgrowingunstopablyomgiamstillgoingonokdone";
+  }
+};
+
+const meta: Meta<typeof TruncatedText> = {
+  component: TruncatedText,
+  title: "Namekit/TruncatedText",
+  args: {
+    text: "",
+    textStylingClasses: "",
+    tooltipTextStylingClasses: "",
+    displayTooltipWhenTextOverflows: true,
+    maxDisplayWidth: undefined,
```

Comment:
@FrancoAguzzi Let's please ensure we never create storybook stories where one of the controls is undefined.

![CleanShot 2024-08-28 at 11 10 20](https://github.com/user-attachments/assets/0df32b0f-5425-486e-b4fc-bc406f062eb1)

Reply:
@FrancoAguzzi I shared a suggestion somewhere else previously (not sure where) that the `Tooltip` UI component should not have a default max width of infinity but instead something that was a reasonable default for lots of cases. Ex: Maybe somewhere between 200 or 400px.

We want to have reasonable defaults so that people only need to override when its a special situation.

There's almost no situation where someone would actually want an infinite width tooltip.

Reply:
As aligned, these select options where replaced by a text field 🧑🏼‍💻👍🏼